x-pack/plugins/enterprise_search/public/applications/workplace_search/constants.ts

```diff
@@ -583,8 +583,8 @@ export const NON_PLATINUM_OAUTH_DESCRIPTION = i18n.translate(
   }
 );
 
-export const NON_PLATINUM_OAUTH_LINK = i18n.translate(
-  'xpack.enterpriseSearch.workplaceSearch.nonPlatinumOauthLinkLabel',
+export const EXPLORE_PLATINUM_FEATURES_LINK = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.explorePlatinumFeatures.link',
```

Comment:
As per the i18n guidelines, can we change this to: 
```suggestion
  'xpack.enterpriseSearch.workplaceSearch.explorePlatinumFeaturesLinkLabel',
```

Reply:
RE: "id format" replied here: https://github.com/elastic/kibana/pull/112136#issuecomment-919985697